kpi/tests/api/v2/test_api_assets.py

```diff
@@ -529,12 +529,58 @@ def test_project_views_for_anotheruser_can_change_metadata(self):
         regional_res = self.client.get(
             results[0]['assets'], HTTP_ACCEPT='application/json'
         )
-        asset_data = regional_res.json()['results'][0]
+        asset_detail_url = regional_res.json()['results'][0]['url']
+        asset_detail_response = self.client.get(asset_detail_url)
+        asset_data = asset_detail_response.data
+
+        # Copy asset properties to test update
+        settings = copy.deepcopy(asset_data['settings'])
+        settings['country'].append(
+            {'value': 'MEX', 'label': 'Mexico'}
+        )
+        summary = copy.deepcopy(asset_data['summary'])
+        summary['languages'].append('Español (es)')
+
+        content = copy.deepcopy(asset_data['content'])
+        content['survey'].append(
+            {
+                'type': 'text',
+                'name': 'q2',
+                'label': 'q2',
+            },
+        )
+
+        data = {
+            'name': 'A new name',
+            'settings': json.dumps(settings),
+            'summary': json.dumps(summary),
+            'content': json.dumps(content),
+        }
+
         change_metadata_res = self.client.patch(
-            asset_data['url'], data={'name': 'A new name'}
+            asset_detail_url, data=data
         )
         assert change_metadata_res.status_code == status.HTTP_200_OK
 
+        # Validate anotheruser can update only `name` and `settings`
+        asset_detail_response = self.client.get(asset_detail_url)
+        asset_data = asset_detail_response.data
+        # `name` and `settings` should have changed
+        assert asset_data['name'] == data['name']
+        # Remove calculated field `country_codes`
+        asset_data['settings'].pop('country_codes')
+        settings.pop('country_codes')
+        assert asset_detail_response.data['settings'] == settings
+        # `summary` and `content` should have not
+        assert asset_data['summary'] != summary
+        assert asset_data['content'] != content
+        assert self._sorted_dict(asset_data['summary']) == self._sorted_dict(
+            asset_data['summary']
+        )
+        assert self._sorted_dict(asset_data['content']) == self._sorted_dict(
+            asset_data['content']
+        )
```

Comment:
@noliveleger these are like `assert x == x` are they not? :monocle_face: Reformatted:
```python
assert self._sorted_dict(asset_data['summary'])
    == self._sorted_dict(asset_data['summary'])

assert self._sorted_dict(asset_data['content'])
    == self._sorted_dict(asset_data['content'])
```

Reply:
Oops :-P  Indeed. Not what I wanted to do. 
That's how you make tests pass no matter what :-P